src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
+
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const sourceArr = sourceString
+    .split(';\n')
+    .filter((string) => string.includes(':'));
+
+  sourceArr.forEach((style) => {
+    const styleArray = style.split(':');
+    const key = styleArray[0].trim();
+    const styleValueSet = new Set(styleArray[1].split(' '));
```

Comment:
Using a Set to remove whitespace might not be the best approach. It could lead to unintended results if the style value contains duplicate valid values. Instead a direct trim operation can remove leading and trailing whitespaces.